vignettes/data-sources.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Data sources"
+output: rmarkdown::html_vignette
+bibliography: references.bib
+csl: vancouver.csl
+vignette: >
+  %\VignetteIndexEntry{Data sources}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  echo = FALSE,
+  results = "asis",
+  collapse = TRUE,
+  comment = "#>"
+)
+library(tidyverse)
+library(osdc)
+```
+
+This document describes the sources of data needed by the OSDC algorithm
+and gives a brief overview of each of these sources and how they might
+look like. The algorithm uses these Danish registers as input data
+sources:
+
+```{r, results='asis'}
+osdc:::registers_as_md_table("Danish registers used in the OSDC algorithm.")
```

Comment:
The triple-colons is because this is an internal function/object or what is going on here?
Also, a note to my future self: you need to build the package in order to access internal objects

Reply:
haah yes! Or use `Ctrl-Shift-L` to load the package. And yea, `:::` accesses all internal objects in a package, neat trick!